test/erc_to_erc/foreign_bridge.test.js

```diff
@@ -29,6 +29,9 @@ contract('ForeignBridge_ERC20_to_ERC20', async (accounts) => {
       '0'.should.be.bignumber.equal(await foreignBridge.deployedAtBlock())
       false.should.be.equal(await foreignBridge.isInitialized())
       '0'.should.be.bignumber.equal(await foreignBridge.requiredBlockConfirmations())
+
+      await foreignBridge.initialize(validatorContract.address, token.address, requireBlockConfirmations).should.be.rejectedWith(INVALID_ARGUMENTS);
```

Comment:
Does it makes sense to cover scenarios?
* `validatorContract` is zero, or it is not a contract address;
* `token` is zero, or it is not a contract address;
* `requireBlockConfirmations` is zero;
* `gasPrice` is zero.

Reply:
Added tests to cover those scenarios except when `validatorContract` and `token` are not contract address. Those kind of checks are not performed on any bridge contract on initialization. Should we open an issue to add those checks?

Reply:
> Should we open an issue to add those checks?

Yes, please open it.